src/ui/map.js

```diff
@@ -57,7 +57,9 @@ const defaultOptions = {
 
     renderWorldCopies: true,
 
-    refreshExpiredTiles: true
+    refreshExpiredTiles: true,
+
+    redoPlacementOnPan: true
```

Comment:
This is just to keep `map.test.js` working -- the style test stubs can't handle `redoPlacement` calls. I'll take another look at updating the stubs.